app/locales/es/translations.js

```diff
@@ -1,11 +1,160 @@
 export default {
-  // "some.translation.key": "Text for some.translation.key",
-  //
-  // "a": {
-  //   "nested": {
-  //     "key": "Text for a.nested.key"
-  //   }
-  // },
-  //
-  // "key.with.interpolation": "Text with {{anInterpolation}}"
+  'site_title': 'datafruits',
+  'site_title_fm': 'datafruits.fm',
```

Comment:
No need to specify text that doesn't change for Spanish, it'll default to the English file when a translation is not found. The site title at least can be left out :)